src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatWidget.ts

```diff
@@ -44,7 +45,7 @@ const enum Message {
 	ReturnInput = 1 << 6,
 }
 
-const terminalChatPlaceholder = localize('default.placeholder', "Ask how to do something in the terminal");
+const terminalChatPlaceholder = localize('default.placeholder', "Ask Copilot");
```

Comment:
We shouldn't hardcode product names here. You can use the chat service to lookup the default agent for your location and use its name.